raop/featureextract/featureextract.py

```diff
@@ -1 +1,31 @@
-#
+import re
+
+class FeatureExtract(object):
+    def  __init__(self):
+        self.evidence = None
+        self.statusKarma = None
+	self.statusAccAge = None
+	self.statusPrevAct = None
+
+    def findEvidence(self,request_text_edit_aware):
+	'''Find reddit post with image/proof'''
+
```

Comment:
Perhaps a sentence in doc string explaining what evidence is and what the re is searching for as evidence